README.md

```diff
@@ -1,104 +1,347 @@
-# Json Schema
+# JSON Schema
 
-  A *dart:io* dependent library for validating json instances against json schema (version Draft 04)
+  A *platform agnostic* (dart:html or dart:io) Dart library for validating JSON instances against JSON Schemas (multi-version support with latest of Draft 6).
 
 ![Build Status](https://travis-ci.org/workiva/json_schema.svg)
 
+## How To Create and Validate Against a Schema
+  
+### Synchronous Creation - Self Contained
+  
+The simplest way to create a schema is to pass JSON data directly to `JsonSchema.createSchema` with a JSON `String`, or decoded JSON via Dart `Map` or `bool`. 
+
+After creating any schema, JSON instances can be validated by calling `.validate(instance)` on that schema. By default, instances are expected to be pre-parsed JSON as native dart primitives (`Map`, `List`, `String`, `bool`, `num`, `int`). You can also optionally parse at validation time by passing in a string and setting `parseJson`: `schema.validate('{ "name": "any JSON object"}', parseJson: true)`.
+    
+    Note: Creating JsonSchemas synchronously implies access to all $refs within the root schema. If you don't have access to all this data at the time of the construction, see "Asynchronous Creation" examples below.
```

Comment:
This line renders with a scrollbar, which doesn't seem great:

![renderwithscrollbar](https://user-images.githubusercontent.com/34247763/48148789-d823a380-e277-11e8-90da-c6c42fa3dbef.png)

If you want this to be set apart somehow, perhaps prefix with `>`?

> Then the text will be in a blockquote, which may be better - at least it will not require users to scroll.

Reply:
great suggestion, thanks! that's closer to what I was looking for anyway